mrn_mysql_compat.h

```diff
@@ -962,3 +962,23 @@ typedef uint mrn_srid;
 #  define MRN_SET_MESSAGE_FROM_CTX(ctx, error_code)     \
   my_message(error_code, ctx->errbuf, MYF(0))
 #endif
+
+#if defined(MRN_MARIADB_P) &&                                      \
+    ((MYSQL_VERSION_ID >= 100526 && MYSQL_VERSION_ID < 100600) ||  \
+     (MYSQL_VERSION_ID >= 100619 && MYSQL_VERSION_ID < 100700) ||  \
+     (MYSQL_VERSION_ID >= 101109 && MYSQL_VERSION_ID < 101200))
+
+#  define MRN_GET_TABLE_NAME(query_tables)                         \
+  (query_tables->get_table_name().str)
+#  define MRN_GET_TABLE_LENGTH(query_tables)                       \
```

Comment:
Could you use `..._TABLE_NAME_LENGTH` instead of `..._TABLE_LENGTH`?

Reply:
I used `..._TABLE_NAME_LENGTH` instead of `..._TABLE_LENGTH`.